configure.php

```diff
@@ -575,6 +601,53 @@ function delete_files( string|array $paths ): void {
 	echo "Done!\n\n";
 }
 
+write( 'Installing Required Plugins...' );
+$required_file_contents = file_get_contents( 'composer-templates/default.json' );
+$required_plugins       = json_decode( $required_file_contents, true );
+foreach( $required_plugins as $plugin ) {
+	install_plugin( $plugin );
+}
+
+write( 'Installing Suggested Plugins...' );
+$suggested_file_contents = file_get_contents( 'composer-templates/suggested.json' );
+$suggested_plugins       = json_decode( $suggested_file_contents, true );
+foreach( $suggested_plugins as $plugin ) {
+	install_plugin( $plugin, true );
```

Comment:
I'd rather this be interactive, where for each suggested plugin it prompts the user as to whether they want to install it or not, and only does so if they say yes.

Reply:
@kevinfodness it does - that's the way this works. The second param on `install_plugin` is `$prompt` if `true`, it asks and only installs if the answer is yes.

Reply:
Great - thanks, missed that detail on my initial review.